rascal-vscode-extension/src/test/vscode-suite/ide.test.ts

```diff
@@ -187,5 +187,28 @@ describe('IDE', function () {
         expect(editorText).to.contain("i - 1");
         expect(editorText).to.contain("i -2");
     });
+
+    it("code actions work", async() => {
+        const editor = await ide.openModule(TestWorkspace.libCallFile);
+        await editor.moveCursor(1,8); // in the module name
+
+        try {
+            // trigger the code actions
+            const inputarea = await editor.findElement(By.className('inputarea'));
+            await inputarea.sendKeys(Key.chord(TextEditor.ctlKey, "."));
+
+            // finds an open menu with the right item in it (Change to a) and then select
+            // the parent that handles UI events like click() and sendKeys()
+            const menuContainer = await ide.hasElement(editor, By.xpath("//div[contains(@class, 'focused') and contains(@class, 'action')]/span[contains(text(), 'Add missing license header')]//ancestor::*[contains(@class, 'monaco-list')]"), Delays.normal, "Add-license action should be available and focused");
```

Comment:
we should refactor this into a function into the IDE class, with the text as an argument.

Reply:
ok will do